diff -ruN orig/wrapper/libvmaf.pc new/wrapper/libvmaf.pc
--- orig/wrapper/libvmaf.pc	2019-09-08 12:35:54.000000000 -0600
+++ new/wrapper/libvmaf.pc	2020-02-18 17:25:37.566081069 -0600
@@ -1,7 +1,7 @@
-prefix=/usr/local
+prefix=/usr
 exec_prefix=${prefix}
-libdir=/usr/local/lib
-includedir=/usr/local/include
+libdir=${prefix}/@libdir@
+includedir=/usr/include
 
 Name: libvmaf
 Description: Netflix's VMAF library
diff -ruN orig/wrapper/Makefile new/wrapper/Makefile
--- orig/wrapper/Makefile	2019-09-08 12:35:54.000000000 -0600
+++ new/wrapper/Makefile	2020-02-18 17:31:54.220195706 -0600
@@ -8,7 +8,16 @@
 OBJDIR = $(TOP)/obj
 FEATURESRCDIR = $(TOP)/../feature/src
 PTOOLSDIR = $(TOP)/../ptools
-INSTALL_PREFIX = /usr/local
+MACHINE := $(shell uname -m)
+
+ifeq ($(MACHINE), x86_64)
+libdir = lib64
+endif
+ifeq ($(MACHINE), i686)
+libdir = lib
+endif
+
+INSTALL_PREFIX = /usr
 INCLUDES += -I$(TOP)/../feature/src
 INCLUDES += -I$(TOP)/../feature/src/common
 
@@ -168,24 +177,24 @@
 
 .PHONY: install
 install:
-	mkdir -p $(DESTDIR)$(INSTALL_PREFIX)/lib
+	mkdir -p $(DESTDIR)$(INSTALL_PREFIX)/$(libdir)
 	mkdir -p $(DESTDIR)$(INSTALL_PREFIX)/include
 	mkdir -p $(DESTDIR)$(INSTALL_PREFIX)/share
-	mkdir -p $(DESTDIR)$(INSTALL_PREFIX)/lib/pkgconfig
-	cp $(LIBVMAF) $(DESTDIR)$(INSTALL_PREFIX)/lib/$(LIBVMAF)
+	mkdir -p $(DESTDIR)$(INSTALL_PREFIX)/$(libdir)/pkgconfig
+	cp $(LIBVMAF) $(DESTDIR)$(INSTALL_PREFIX)/$(libdir)/$(LIBVMAF)
 	cp src/libvmaf.h $(DESTDIR)$(INSTALL_PREFIX)/include/
 	cp -r ../model $(DESTDIR)$(INSTALL_PREFIX)/share/
-	cp libvmaf.pc $(DESTDIR)$(INSTALL_PREFIX)/lib/pkgconfig/
+	cp libvmaf.pc $(DESTDIR)$(INSTALL_PREFIX)/$(libdir)/pkgconfig/
 
 .PHONY: uninstall
 uninstall:
-	rm -f $(DESTDIR)$(INSTALL_PREFIX)/lib/$(LIBVMAF)
+	rm -f $(DESTDIR)$(INSTALL_PREFIX)/$(libdir)/$(LIBVMAF)
 	rm -f $(DESTDIR)$(INSTALL_PREFIX)/include/libvmaf.h
 	rm -fr $(DESTDIR)$(INSTALL_PREFIX)/share/model
-	rm -f $(DESTDIR)$(INSTALL_PREFIX)/lib/pkgconfig/libvmaf.pc
+	rm -f $(DESTDIR)$(INSTALL_PREFIX)/$(libdir)/pkgconfig/libvmaf.pc
 
 testlib: $(SRCDIR)/main.cpp
-	$(CXX) -s -o $@  $(CXXFLAGS) $^ -I $(FEATURESRCDIR) -I $(FEATURESRCDIR)/common -I $(DESTDIR)$(INSTALL_PREFIX)/include -L$(DESTDIR)$(INSTALL_PREFIX)/lib -lvmaf -pthread
+	$(CXX) -s -o $@  $(CXXFLAGS) $^ -I $(FEATURESRCDIR) -I $(FEATURESRCDIR)/common -I $(DESTDIR)$(INSTALL_PREFIX)/include -L$(DESTDIR)$(INSTALL_PREFIX)/$(libdir) -lvmaf -pthread
 
 .PHONY: clean
 clean:
